src/graph/commands/histogram/histogram.pas

```diff
@@ -0,0 +1,211 @@
+unit histogram;
```

Comment:
A lot of the comments from barchart also apply here. Especially in relations to command options and potential memory leaks.